source/help/apps/desktop-guide.rst

```diff
@@ -0,0 +1,187 @@
++-------------------------------------------------------------------------------+---------------------------+---------------------------+---------------------------+
+| Start app on login                                                            | v1.3.0 +                  | v1.3.0 +                  | v1.3.0 +                  |
++-------------------------------------------------------------------------------+---------------------------+---------------------------+---------------------------+
+| Toggle window visibility when clicking on the tray icon                       | v1.3.0 +                  | N/A                       | N/A                       |
++-------------------------------------------------------------------------------+---------------------------+---------------------------+---------------------------+
+| Leave app running in notification center when application window is closed    | N/A                       | v1.3.0 +                  | v1.3.0 +                  |
++-------------------------------------------------------------------------------+---------------------------+---------------------------+---------------------------+
+| Show red badge on taskbar icon to indicate unread messages                    | v1.4.0 +                  | v1.4.0 +                  | N/A                       |
++-------------------------------------------------------------------------------+---------------------------+---------------------------+---------------------------+
+| Flash taskbar icon when a new message is received                             | v1.3.0 +                  | N/A                       | v1.4.0 +                  |
++-------------------------------------------------------------------------------+---------------------------+---------------------------+---------------------------+
+
+Hide menu bar
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Hides the menu bar. When enabled, pressing ``ALT`` will toggle whether the menu bar is shown or hidden.
+
+This setting is off by default.
+
+Allow mixed content
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+If your server is hosted on `https://`, insecure content, images and scripts with `http://` are not rendered by default. This option allows such content to be rendered. If you do enable it, please be mindful of potential security risks shared over `http://` protocols.
+
+This setting is on by default.
+
+Note: Enabling mixed content will disable YouTube video previews [due to an issue in the underlying technology](https://github.com/electron/electron/issues/2749) used by the Mattermost Desktop app.
+
+Show icon on Menu Bar (OS X only)
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+When enabled, a black dot indicating unread messages is displayed on the team tab bar.
```

Comment:
I believe this featured is updated - "Team tab is bolded for unread messages and has a red dot with a count of unread mentions?"